swagger.yaml

```diff
@@ -6865,6 +6865,7 @@ definitions:
       - 'mce-requirements-satisfied'
       - 'network-type-valid'
       - 'platform-requirements-satisfied'
+      - 'cluster-preparation-succeeded'
```

Comment:
what is the meaning of this new validation? in which wizard step the UI will display it?

Reply:
I will remove this one as well, thanks